lib/ansible/modules/network/ios/ios_banner.py

```diff
@@ -135,9 +132,6 @@ def map_config_to_obj(module):
 
 def map_params_to_obj(module):
     text = module.params['text']
```

Comment:
any specific reason of removing the same

Reply:
again we don;t want to strip those intentional whitespace characters .

Reply:
+1